components/financial_assistance/app/models/financial_assistance/services/summary_service.rb

```diff
@@ -175,7 +178,7 @@ def load_coverages_map(hash, kind)
                 return unless has_kind
 
                 coverage_map = ApplicantCoverageConfigLoader.new(@applicant, kind).load_config
-                hash[:health_coverage][:rows][kind][:coverages] = coverage_map
```

Comment:
I would look into using dig here when going this deep into a hash

Reply:
I've updated this method to safely read from and write to this hash.